corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js

```diff
@@ -505,35 +505,6 @@ hqDefine('app_manager/js/forms/case_config_ui', function () {
                 return 'commcare-user';
             };
 
-            self.setRequired = function (required) {
```

Comment:
same question I have about everything else. was this not used?

Reply:
weird diff